<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewClauseDialog</class>
 <widget class="QDialog" name="NewClauseDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>202</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Nova Cláusula</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="newClauseLabel">
     <property name="text">
      <string>Nova Cláusula</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="titleLabel">
       <property name="text">
        <string>Título</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="titleEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="documentLabel">
       <property name="text">
        <string>Documento:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="typeLabel">
       <property name="text">
        <string>Tipo:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="parentLabel">
       <property name="text">
        <string>Cláusula Pai:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="parentBox"/>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="documentBox"/>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="typeBox"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NewClauseDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>180</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>201</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NewClauseDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>186</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>201</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
